context.go

```diff
@@ -32,7 +31,10 @@ import (
 	"time"
 
 	"github.com/Unknwon/com"
+	"golang.org/x/crypto/pbkdf2"
 
+	"crypto/rand"
```

Comment:
Please move line 36, 37 to after line 31, so standard libraries are grouped together.